tools/c7n_azure/c7n_azure/resources/storage.py

```diff
@@ -47,6 +60,42 @@ class resource_type(ArmResourceManager.resource_type):
 
 @Storage.action_registry.register('set-network-rules')
 class StorageSetNetworkRulesAction(AzureBaseAction):
+    """ Set Network Rules Action
+
+    Updates Azure Storage Firewalls and Virtual Networks settings.
+
+    :example:
+
+    Find storage accounts without open ip list and restrict them.
```

Comment:
We should rephrase this description.
"Restrict access to storage accounts that do not have any ip rules."
"Bypass these rules for logging and metrics."